mcs/resource_mcs_db_user_test.go

```diff
@@ -201,7 +201,7 @@ resource "mcs_db_database" "testdb2" {
 resource "mcs_db_user" "basic" {
   name        = "basic"
   instance_id = "${mcs_db_instance.basic.id}"
-  password    = "password"
+  password    = "Qw!weZ12$"
```

Comment:
🤨

Reply:
More strong password is required, otherwise user creation will fail